bundles/org.openhab.persistence.influxdb/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,105 @@
+			<advanced>false</advanced>
+		</parameter-group>
+
+		<parameter name="url" type="text" required="true" groupName="connection">
+			<context>url</context>
+			<label>Database URL</label>
+			<description>The database URL, e.g. http://127.0.0.1:8086 or http://127.0.0.1:9999</description>
+			<default>http://127.0.0.1:8086</default>
+		</parameter>
+
+		<parameter name="version" type="text" required="true" groupName="connection">
+			<label>Database version</label>
+			<description>InfluxDB version</description>
+			<default>V1</default>
+			<options>
+				<option value="V1">InfluxDB 1</option>
+				<option value="V2">InfluxDB 2</option>
+			</options>
+		</parameter>
+
+		<parameter name="user" type="text" required="true" groupName="connection">
+			<label>Username</label>
+			<description>Database username</description>
+			<default>openhab</default>
+		</parameter>
+
+		<parameter name="password" type="text" required="false" groupName="connection">
+			<context>password</context>
+			<label>Database password</label>
+			<description>Database password</description>
+		</parameter>
+
+		<parameter name="token" type="text" required="false" groupName="connection">
+			<label>Authentication token</label>
+			<description>The token to authenticate to database (alternative to username/password for InfluxDB 2.0)
+			</description>
+		</parameter>
+
+		<parameter name="db" type="text" required="true" groupName="connection">
+			<label>Database/Organization</label>
+			<description>The name of the database (InfluxDB 1.0) or Organization for (InfluxDB 2.0)</description>
+			<default>openhab</default>
+		</parameter>
+
+		<parameter name="retentionPolicy" type="text" required="true" groupName="connection">
+			<label>Retention policy / Bucket</label>
+			<description>The name of the retention policy (Influx DB 1.0) or bucket (InfluxDB 2.0) to write data
+			</description>
+			<default>openhab</default>
+		</parameter>
+
+		<parameter name="replaceUnderscore" type="boolean" required="true" groupName="misc">
+			<label>Replace underscore</label>
+			<description>Whether underscores "_" in item names should be replaced by a dot "." ("test_item" ->
+				"test.item"). Only for measurement name, not for tags. Also applies to alias names.
+			</description>
+			<default>false</default>
+		</parameter>
+
+		<parameter name="addCategoryTag" type="boolean" required="true" groupName="tags">
+			<label>Add category tag</label>
+			<description>Should the category of the item be included as tag "category"? If no category is set, "n/a" is
+				used.
+			</description>
+			<default>false</default>
+		</parameter>
+
+		<parameter name="addTypeTag" type="boolean" required="true" groupName="tags">
+			<label>Add type tag</label>
+			<description>Should the item type be included as tag "type"?</description>
+			<default>false</default>
+		</parameter>
+
+		<parameter name="addLabelTag" type="boolean" required="true" groupName="tags">
+			<label>Add label tag</label>
```

Comment:
The first letter of every word in a label should be capitalized. Please update other labels as well.
```suggestion
			<label>Add Label Tag</label>
```